unix/w0vncserver/portals/RemoteDesktop.cxx

```diff
@@ -499,3 +526,134 @@ bool RemoteDesktop::parseStreams(GVariant* streams)
 
   return true;
 }
+
+bool RemoteDesktop::loadRestoreToken()
+{
+  char filepath[PATH_MAX];
+  FILE* f;
+  const char* stateDir;
+  char restoreToken_[37];
+
+  // Allows users to reset the restore token by not choosing "Never"
```

Comment:
That's a bit convoluted. A small script helper to reset the token would be preferred.